src/Miunie.Core/IMiunieDiscord.cs

```diff
@@ -6,6 +6,7 @@ namespace Miunie.Core
 {
     public interface IMiunieDiscord
     {
+        ulong GetBotId();
```

Comment:
Since `Bot` directly references Miunie herself, I think it would be better to communicate what we're doing a little bit more. So instead of generally getting the `ulong`  Id, we can have a method like:

```cs
bool UserIsMiunie(MiunieUser user);
```

Right? 😊 

That way the way you're calling it gets simplified **and** we more clearly say what we're doing.

What do you think? 🙂

Reply:
Sounds good to me. I will make this change.